### PR TITLE
Unfuckups the fuck-up boots

### PR DESCRIPTION
**Pull request by ShiftyRail**
Higher effort PR than #26117.

- Fuckup boots no longer breach as you walk
- When equipping fuckup boots, after walking with them for a while (in order to prevent exploits to automatically reset the cooldown), you can *activate* them to fuck up the ground
- The activation spell has a 30 seconds cooldown
- Ground destruction last for **4** steps or **3** seconds (the shortest applies)
- When you blink, the cooldown for the activation spell go up by at most 4 seconds
- Walk delay while the boots are active reduced from **1.5 seconds to 1 second** (it's quite noticeable)

## Why

It tweaks the content to be more fun to use and less unfun to play against instead of reverting it.

:cl:
- tweak: The fuckup boots will no longer breach as you walk. You will need to activate them before destroying the ground. This doesn't actually add them back to the spellbook.

### CODE REVIEW

**Comment by aceedex** [comment]
What does it mean to walk with them on for a while?
The  most obnoxious thing about them is being able to use blink and jaunt imo. While this is high effort I don't think it adressed the big issue with em

**Comment by ShiftyRail** [comment]
They can only be used after 50 footsteps (which sounds like a lot, but in fact you can do this in 20 seconds).
Good point on jaunt, stand back for more changes

**Comment by aceedex** [comment]
Should also make the spell require robes if it doesn't. I'm not too privy with how the boots work but it would be dumb if you could jaunt in to secure engi storage, grab a suit and just go wild with the suit hiding the boots

**Comment by cloutcode** [comment]
like i've said before if someone blinks with the fuckup boots on have it rip both their legs off instantly

**Comment by ShiftyRail** [comment]
@cloutcode I don't think it's very fun to micromanage your inventory. It's funnier to try the combination of both spells and see if it works or not.

I've changed the ~balance~ a bit :

- When you cast the activation spell, you shout an identifiable phrase and your boots instantly start to make noise
- After 4 footsteps (which is short, but enough to react if you're trained), the ground starts to fuck up and the wizard is slowed down
- Using blink or ethereal jaunt at any time instantly cancels the spells.

In short, you can still use blink to safely retreat, but your opponents have time to react before the breaching happens, and if you need to get away fast, you give up the breaching. I think it's fair for both sides.

**Comment by SonixApache** [comment]
polishing this turd will not provide any benefit

**Comment by TGFH** [comment]
I'd much rather see them go the way of the dodo and get replaced with something totally different at a later date.

**Comment by Kurfursten** [comment]
Currently the other PR has a higher proportion of support (73% vs 71%). I'll give it one more day.

**Comment by ShiftyRail** [comment]
2% difference isn't significant

If you want to truly gauge opinions, make a poll with **multiple** answers and choose the answer with the highest support

**Comment by RealestEstate** [comment]
It's gonna be hilarious when the boots deactivate and the wizard gets sucked into the breach he/she just made

**Comment by ghost** [comment]
no thumbs up until teleporting with them active rips off legs

**Comment by ShiftyRail** [comment]
@ancientpower not fun at all for the wiz to be punished like that

**Comment by jknpj** [comment]
>not fun at all for the wiz to be punished like that

Good. Blinkspam is for fags

**Comment by Kurfursten** [comment]
Why not just make it so the blink won't go off when you're anchored by mags instead of making a newbie trap?

"Haha you never tried this before, so your round is over"

**Comment by Kurfursten** [comment]
Since you haven't closed this, and it remains popular, I see no reason why it can't also be merged. But I added to your changelog.
